.eslintrc.yaml

```diff
@@ -9,3 +9,7 @@ parserOptions:
 
 extends:
   - prettier
+
+rules:
+  no-var: "warn"
+  object-shorthand: "warn"
```

Comment:
Added the `no-var` and `object-shorthand` rules to eslint.

This'll be run by precommit over staged files, and issues (if any) will be fixed automatically.

Reply:
Thanks for doing this !!